zksync/.env.mainnet.example

```diff
@@ -0,0 +1,55 @@
+CHAIN_ETH_NETWORK=mainnet
```

Comment:
In our deploy.sh script we do not put the contract implementation addresses to the env file, only proxies. So with this and deployment acceptance tests you added, we should just keep in mind that the env would need to be manually updated with implementation addresses.